### ISSUE TITLE
Overrides default auto completion

### ISSUE THREAD

**Issue by markgoodyear** [liamcain/AutoFileName#18]
When using this plugin it seems to override the default auto completion. For example, when typing `<div class="` it will bring up the file name completion, rather than previous class names. 

I'm not sure if I'm doing anything wrong, but my set up is pretty much the standard Sublime Text 2 install on Mac with a few plug-ins.

Cheers

**Comment by iirvine** [comment]
I'm also getting this, also on a Mac. Seems the plugin will bring up a file autocomplete box on any string typed in any file, which gets a little annoying working with non-HTML files... The plugin is quite handy, but I don't really want it popping up _every_ time I type a string.

**Comment by scarlight** [comment]
Yes I'm having the same problem as markgoodyear, this plugin takes over the sublime native autocomplete feature as well as another useful plugin called "All Autocomplete" that extends native autocompletion from other opened files :(

**Comment by markbailey1980** [comment]
I'm also experiencing this issue on Mac.

**Comment by bachue** [comment]
I would rather trigger its autocomplete by ctrl-x, ctrl-f, just like we did in Vim, but still use default autocompletion as usual.

**Comment by liamcain** [comment]
I added a new setting if it gets in the way too frequently, you can set it to a keybinding so it doesn't get added into the default completion results. However, I consider this only a temporary fix. I plan on reducing the number of situations that it appears in and hopefully preventing it from removing the other completions.

These changes are only in the st3 branch.

**Comment by gonzaloelipse** [comment]
Perfect, I was about to report this issue but now I see some already did it. Waiting for the fix, great work!

**Comment by christierney402** [comment]
I would love to see this fix in a current stable release since it's taking awhile for ST3 to get released.

**Comment by rizzomichaelg** [comment]
any updates to this? 
@evandrocoan any thoughts on fixing this in your fork?

**Comment by evandrocoan** [comment]
I have been thinking about merger the Sublime Completions with this packages completions, but I have forgot to work on it.  

Now looking over my forks code, it seems already doing it. You only need to comment out these lines:
```python
File: Data/Packages/AutoFileName/autofilename.py
280:         if ( uses_keybinding and file_name ) and not FileNameComplete.is_active:
281:             return
```
-->
```python
File: Data/Packages/AutoFileName/autofilename.py
280:         # if ( uses_keybinding and file_name ) and not FileNameComplete.is_active:
281:         #    return
```
But I still not sure how the plugin will work. I will start using it like this to see what happens.

**Comment by rizzomichaelg** [comment]
That seems to have done the trick for me.  I'll keep experimenting with it to see if there's any unexpected behavior.

**Comment by evandrocoan** [comment]
It is a bug on Sublime Text core:

1. https://github.com/SublimeTextIssues/Core/issues/819 on_query_completions fails to return custom completions when some characters are used

But I released some attempted fix to overcome this on my fork.
